src/public/services/__tests__/AdminService.test.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import MockAxios from 'jest-mock-axios'
+import { UiCookieValues } from 'shared/types'
+
+import { adminChooseEnvironment } from '../AdminService'
+
+jest.mock('axios', () => MockAxios)
+
+describe('AdminService', () => {
+  describe('adminChooseEnvironment', () => {
+    const MOCK_UI = UiCookieValues.React
+    const MOCK_RESPONSE = UiCookieValues.React
+
+    it('should return environment name if GET request succeeds', async () => {
+      // Arrange
+      const mockUi = UiCookieValues.React
+      const expectedResponse = UiCookieValues.React
+
+      // Act
+      const actual = adminChooseEnvironment(MOCK_UI)
+      MockAxios.mockResponse({ data: MOCK_RESPONSE })
+
+      // Assert
+      await expect(actual).resolves.toEqual(MOCK_RESPONSE)
```

Comment:
Is this assertion meaningful to the test? It's looks like it's testing `MockAxios` rather than our own code.

Reply:
removed `MockAxios.mockResponse`